Application/src/test/java/io/github/antijava/marjio/network/PackerTest.java

```diff
@@ -0,0 +1,128 @@
+package io.github.antijava.marjio.network;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Date on 2015/12/29.
+ */
+public class PackerTest {
+
+    private StatusData player;
+    private StatusData block;
+    private StatusData item;
+
+    private final int PLAYER = 0;
+    private final int BLOCK = 1;
+    private final int ITEM = 2;

```

Comment:
Change to use `StatusData.Player if [final int](https://github.com/NCU-Anti-Java/Marjio/pull/48/files#r48528184) be `static`?